server/backbone/notifications.py

```diff
@@ -0,0 +1,52 @@
+from pyfcm import FCMNotification
+from abc import ABC, abstractmethod
+from .models import Parent, Child
+
+
+class Notification(ABC):
+
+  api_key = "AAAAuKgxco4:APA91bF0GiF-EGkdRLYXzZYIf76YQffw_93_GTut2N2pj48CS_Vdw44scWBzJqkobei5xcfE9IET2mXLIOGQJdOGQbhkal_A9BYfQNQ3L1AWcZnsIQdfoiku2htuETMOB2qzeehBQ2hE"
```

Comment:
is this ok lol

Reply:
No, not in my code!
Create a new file, next to `secret_key.txt` and set a variable in `/backbone/__init__.py` just like how I did with `project_id`. Then you can see how I access that value in the function, `parse_quest_test`. Instead of making this file required, make the complete and verify end-points only optionally send notifications.